src/types.ts

```diff
@@ -63,19 +63,29 @@ export interface CookieConsentHookActions {
   acceptAllCookies: () => void;
   didAcceptAll: () => boolean;
   didDeclineAll: DidDeclineAllHandler;
-  cookies: CookieWrapper;
+  cookies: CookieWrapperType;
```

Comment:
If the `storage` prop is passed through the props there shouldn't be a `cookies` property at all. The returned property should either be called `storage` or not be at all, which means you need to define another interface for that - don't forget that when re-defining the interface you can use a "Base" interface with all the common properties and then extend two unique return types from it.